packages/nextjs/app/builders/0x167142915AD0fAADD84d9741eC253B82aB8625cd/page.tsx

```diff
@@ -0,0 +1,88 @@
+import Image from "next/image";
+import { Address } from "../../../components/scaffold-eth";
+import { AiFillGithub, AiFillInstagram, AiFillLinkedin, AiFillTwitterCircle } from "react-icons/ai";
+
+const socials = [
+  {
+    name: "GitHub",
+    url: "https://github.com/nnennaokoye",
+    icon: AiFillGithub,
+  },
+  {
+    name: "Twitter",
+    url: "https://x.com/_0xNina",
+    icon: AiFillTwitterCircle,
+  },
+  {
+    name: "LinkedIn",
+    url: "https://www.linkedin.com/in/nnennaokoye/",
+    icon: AiFillLinkedin,
+  },
+  {
+    name: "Instagram",
+    url: "https://www.instagram.com/_ninacodes",
+    icon: AiFillInstagram,
+  },
+];
+
+export default function NnennaOkoyePage() {
```

Comment:
You might want to consider typing your page as NextPage since we are using TypeScript, as this ensures better integration and adds clarity for future maintainers by indicating that the component is a Next.js page.